src/hardware/rdtech-dps/protocol.c

```diff
@@ -396,6 +409,10 @@ SR_PRIV int rdtech_dps_get_state(const struct sr_dev_inst *sdi,
 		is_reg_cc = reg_state == MODE_CC;
 		out_state = read_u16be_inc(&rdptr); /* ENABLE */
 		is_out_enabled = out_state != 0;
+		if (devc->model->id == 60125) {
```

Comment:
I think magic number  "60125" deserves a macro name....

Reply:
Well, in the general case I of course agree that there should not be magic numbers in the code. The reason why I did it this way in this case is that 60125 is not magic but is the most specific name of the model. Thus, a suitable macro name could be MODEL_NUMBER_60125 which is kind of pointless.

What would you say about defining a macro (in protocol.c) containing 60125 but used in this location as:

  `if (IS_MODEL_6012P (devc->model))`

?